src/ArrayFilter.ts

```diff
@@ -0,0 +1,37 @@
+import { buildFilter } from "./Builder";
+import { FilterParams, Filter } from "./Filter";
+import { ArrayFilterOperation } from "./FilterOperation";
+
+
+export interface ArrayFilterParams extends FilterParams {
+  op: ArrayFilterOperation;
+  value: string;
+}
+
+export class ArrayFilter extends Filter implements ArrayFilterParams {
+  public op: ArrayFilterOperation;
+  public value: string;
+  constructor(params: ArrayFilterParams) {
+    super(params);
+    this.op = params.op;
+    this.value = params.value;
+  }
+  public get innerFilterBuild(): string {
+    // example > "o=Any&p=Tests&v=(o=contains&p=Name&v=1,2,3,4,5,6)"
+    const o = `o=${this.op}`;
+    const p = "p=_"
```

Comment:
This shouldn't be a const. This filter should be able to form queries towards complex models as well like below
`o=any&p=myCollection&v=(o=in&p=myCollectionPropertyName&v=123)`
notice that i can also search `where myCollection.PropertyName = 123` with the query above.